pom.xml

```diff
@@ -263,7 +263,7 @@
     <dependency>
       <groupId>com.laserfiche</groupId>
       <artifactId>lf-api-client-core</artifactId>
-      <version>1.0.0</version>
```

Comment:
Is this published to sonar type snapshots?

Reply:
yes -> https://s01.oss.sonatype.org/content/repositories/snapshots/com/laserfiche/lf-api-client-core/1.0.0-preview-3274168162-SNAPSHOT/

Reply:
Can we keep it 1.0.0 in the 1.x branch? The reason is, we have decided to not to work on the pipeline and focus on providing everything for the Integration team. Keeping the version in 1.x as 1.0.0 (so not using the snapshot) would be easier since we haven't resolved the firewall issue (again, we decided to postpone it for support the Integration team).